ui/app/src/App.tsx

```diff
@@ -54,6 +61,9 @@ function App() {
     return Math.random().toString(36).substring(7);
   };
 
+  const { address } = useAccount();
+  const [streamId, setStreamId] = useState<number | null>(null);
```

Comment:
would be cool to use the SDK's `StreamManager.onStreamCreated` function to install the `StreamStatus` items into the list of streams to review on the main demo page (`{demo-app-url}/{StreamManager-address}/all`)

Reply:
OK; I'll create an issue and address this in a separate PR